layers/+lang/go/config.el

```diff
@@ -36,3 +36,6 @@
 
 (defvar go-use-test-args ""
   "Additional arguments to be supplied to `go test` during runtime.")
+
+(defvar go-install-after-save nil
+  "Run `go install --buildmode=archive` after save.")
```

Comment:
Please add the possible values to this docstring as well.

Reply:
ok, fixed